Address build warnings in ncurses example

Just a self-reminder really.
```
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp: In function ‘void drawTopology()’:
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:219:56: warning: zero-length gnu_printf format string [-Wformat-zero-length]
  219 |     mvwprintw(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, "");
      |                                                        ^~
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:241:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  241 |                     int y = 0;
      |                         ^
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:261:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  261 |                     int y = 0;
      |                         ^
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:281:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  281 |                     int y = 0;
      |                         ^
```

> ```
> /home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:219:56: warning: zero-length gnu_printf format string [-Wformat-zero-length]
>   219 |     mvwprintw(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, "");
>       |                                                        ^~
> ```

I'm a little fuzzy on curses API. I think this warning implies that we could simply move the cursor to a position inside the specified "window" using [`mvwaddch()`](https://invisible-island.net/ncurses/man/curs_addch.3x.html) instead of a printf-like function.
```diff
-    mvwprintw(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, "");
+    mvwaddch(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, 0);
```

--- a/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp
+++ b/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp
@@ -56,7 +56,6 @@ int main()
         return 0;
     }
     radio.printDetails();
-    mesh.setStaticAddress(8, 01);
     win = initscr(); /* Start curses mode 		  */
     getmaxyx(win, maxX, maxY);
     start_color();

--- a/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp
+++ b/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp
@@ -8,10 +8,13 @@
  * Requirements: NCurses
  * Install NCurses: apt-get install libncurses5-dev
  * Setup:
- * 1: make
- * 2: sudo ./RF24Mesh_Ncurses_Master
+ * 1. cd ~/RF24Mesh/examples_RPi
+ * 2. mkdir build && cd build
+ * 3. cmake ..
+ * 4. make
+ * 5. ./ncurses/RF24Mesh_Ncurses_Master
  *
- * NOTE: DEBUG MUST BE DISABLED IN RF24Mesh_config.h
+ * NOTE: RF24MESH_DEBUG MUST BE DISABLED IN RF24Mesh_config.h
  *
  * Once configured and running, the interface will display the header information, data rate,
  * and address assignments for all connected nodes.*

--- a/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp
+++ b/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp
@@ -216,7 +216,7 @@ void drawTopology()
     wclear(topoPad);
     wattroff(topoPad, COLOR_PAIR(1));
     mvprintw(17, 10, "Mesh Topology");
-    mvwprintw(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, "");
+    mvwaddch(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, 0);
     wattron(topoPad, COLOR_PAIR(1));
     int topoPadmaxX;
     topoPadmaxX = getmaxx(topoPad);
@@ -238,9 +238,7 @@ void drawTopology()
 
             for (int j = 0; j < mesh.addrListTop; j++) {
                 if (mesh.addrList[j].address == i) {
-                    int y = 0;
-                    int x = 0;
-                    getyx(topoPad, y, x);
+                    int x = getcurx(topoPad);
                     if (x >= topoPadmaxX) wprintw(topoPad, "\n");
                     wprintw(topoPad, "0%o[%d] ", mesh.addrList[j].address, mesh.addrList[j].nodeID);
                 }
@@ -258,9 +256,7 @@ void drawTopology()
 
             for (int j = 0; j < mesh.addrListTop; j++) {
                 if (mesh.addrList[j].address == i) {
-                    int y = 0;
-                    int x = 0;
-                    getyx(topoPad, y, x);
+                    int x = getcurx(topoPad);
                     if (x >= topoPadmaxX) wprintw(topoPad, "\n");
                     wprintw(topoPad, "0%o[%d] ", mesh.addrList[j].address, mesh.addrList[j].nodeID);
                 }
@@ -278,9 +274,7 @@ void drawTopology()
 
             for (int j = 0; j < mesh.addrListTop; j++) {
                 if (mesh.addrList[j].address == i) {
-                    int y = 0;
-                    int x = 0;
-                    getyx(topoPad, y, x);
+                    int x = getcurx(topoPad);
                     if (x >= topoPadmaxX) wprintw(topoPad, "\n");
                     wprintw(topoPad, "0%o[%d] ", mesh.addrList[j].address, mesh.addrList[j].nodeID);
                 }

